package/gluon-status-page/files/lib/gluon/status-page/www/index.html

```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
-<html lang="">
+<html lang="en">
 	<head>
-		<meta http-equiv="Cache-Control" content="no-cache, no-store, must-revalidate">
-		<meta http-equiv="Pragma" content="no-cache">
-		<meta http-equiv="Expires" content="0">
```

Comment:
What is the replacement for these tags, if we want to prevent caching of the redirect?